flexmeasures/api/dev/sensor_data.py

```diff
@@ -0,0 +1,26 @@
+from webargs.flaskparser import use_args
+
+from flexmeasures.data.config import db
+from flexmeasures.data.models.time_series import TimedBelief
+from flexmeasures.api.common.schemas.sensor_data import SensorDataSchema
+
+
+@use_args(
+    SensorDataSchema(),
+    location="json",
+)
+def post_data(sensor_data):
+    """POST to /sensorData
```

Comment:
The API endpoint docstring could also include an example (copied from one of the tests) and possible error codes returned, to prepare for adding the endpoint to the documentation.